code/obj/item/staffkiosk.dm

```diff
@@ -0,0 +1,287 @@
+	name = "Robotics Assistant"
+	accessParent = "Roboticist"
+	canAuthorize = list("Roboticist","Medical Director","Head of Personnel","Captain")
+	cardIcon = "id_res"
+	tokenIcon = "assist-token-robo"
+	dispensedKit = /obj/item/storage/box/staffkit/robo
+
+/datum/recruitment_role/engineer
+	name = "Engineering Assistant"
+	accessParent = "Engineer"
+	canAuthorize = list("Engineer","Chief Engineer","Head of Personnel","Captain")
+	cardIcon = "id_eng"
+	tokenIcon = "assist-token-eng"
+	dispensedKit = /obj/item/storage/box/staffkit/eng
+
+/datum/recruitment_role/mechanic
+	name = "Mechanics Assistant"
+	accessParent = "Mechanic"
+	canAuthorize = list("Mechanic","Chief Engineer","Head of Personnel","Captain")
+	cardIcon = "id_eng"
+	tokenIcon = "assist-token-mech"
+	dispensedKit = /obj/item/storage/box/staffkit/mech
+
+/datum/recruitment_role/mining
+	name = "Mining Assistant"
+	accessParent = "Miner"
+	canAuthorize = list("Miner","Chief Engineer","Head of Personnel","Captain")
+	cardIcon = "id_eng"
+	tokenIcon = "assist-token-mine"
+	dispensedKit = /obj/item/storage/box/staffkit/miner
+
+/datum/recruitment_role/sci
+	name = "Research Assistant"
+	accessParent = "Scientist"
+	canAuthorize = list("Scientist","Research Director","Head of Personnel","Captain")
+	cardIcon = "id_res"
+	tokenIcon = "assist-token-sci"
+	dispensedKit = /obj/item/storage/box/staffkit/res
+
+/datum/recruitment_role/chef
+	name = "Culinary Assistant"
+	accessParent = "Chef"
+	canAuthorize = list("Chef","Head of Personnel","Captain")
+	cardIcon = "id_civ"
+	tokenIcon = "assist-token-chef"
+	dispensedKit = /obj/item/storage/box/staffkit/chef
+
+/datum/recruitment_role/bar
+	name = "Bar Assistant"
+	accessParent = "Bartender"
+	canAuthorize = list("Bartender","Head of Personnel","Captain")
+	cardIcon = "id_civ"
+	tokenIcon = "assist-token-bar"
+	dispensedKit = /obj/item/storage/box/staffkit/bar
+
+/datum/recruitment_role/botany
+	name = "Botanical Assistant"
+	accessParent = "Botanist"
+	canAuthorize = list("Botanist","Head of Personnel","Captain")
+	cardIcon = "id_civ"
+	tokenIcon = "assist-token-hyd"
+	dispensedKit = /obj/item/storage/box/staffkit/botany
+
+
+//equipment kits
+
+/obj/item/storage/box/staffkit/med
+	name = "medical assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffmed,
+		/obj/item/device/radio/headset/medical,
+		/obj/item/disk/data/cartridge/medical,
+		/obj/item/reagent_containers/emergency_injector/epinephrine,
+		/obj/item/reagent_containers/mender/brute,
+		/obj/item/reagent_containers/mender/burn,
+		/obj/item/bandage)
+
+/obj/item/storage/box/staffkit/robo
+	name = "robotics assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffmed,
+		/obj/item/device/radio/headset/medical,
+		/obj/item/disk/data/cartridge/medical,
+		/obj/item/screwdriver,
+		/obj/item/cable_coil,
+		/obj/item/body_bag)
```

Comment:
Body bag?

Reply:
For retrieval of cyborg candidates. Could probably swap it for something else, but not sure what (if removing it, there are up to two slots open).